flink-end-to-end-tests/run-single-test.sh

```diff
@@ -48,6 +48,8 @@ fi
 
 source "${END_TO_END_DIR}/test-scripts/test-runner-common.sh"
 
+cleanup_tmp_files
```

Comment:
commit message is a bit weird, and the description isn't separated from the title